README.md

```diff
@@ -122,6 +122,31 @@ This will output system info such as your NodeJS version, NPM version,
 `zkapp-cli` version, etc. Please include this if submitting a bug report for
 `zkapp-cli`, for easier troubleshooting.
 
+## Contributing
+
+The best way to contribute to the zkApp CLI is to help us test it broadly to
+ensure it works as expected on all platforms. If you encounter any issues,
+please submit an issue on Github and include the info printed when running `zk
+system`, which contains your OS, NodeJS, & zkapp-cli versions to help us
```

Comment:
Just wanted to raise that detecting the SnarkyJS version is still not working as expected. :/ 

This would be really helpful to have when users are submitting issues, maybe we should raise this as a priority to do soon?

Reply:
Good idea